### PR TITLE
Fix #9706, created signing script and added to build.sh

### LINKED ISSUE #9706: Use this script to generate macOS signing certificate
https://github.com/derekparker/delve/blob/a8d11f02a85e99144111afb784cde059ea3faab1/scripts/gencert.sh

instead of pointing the user to a boring interactive guide to do the steps by hand clicking on widgets

### PR DESCRIPTION
**Pull request by PaoloMonti42**

### CODE REVIEW

File: sys/macos-cert.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```
Comment (radare) [inline sys/macos-cert.sh]: use /bin/sh, this script have no bashisms inside

File: sys/build.sh [hunk 2]
```diff
@@ -95,6 +95,7 @@ unset DEPS
 ./configure ${CFGARG} --prefix=${PREFIX} || exit 1
 ${MAKE} -s -j${MAKE_JOBS} MAKE_JOBS=${MAKE_JOBS} || exit 1
 if [ "`uname`" = Darwin ]; then
+	./sys/macos-cert.sh
```
Comment (radare) [inline sys/build.sh]: we dont want to generate a new cert on every run

Comment (PaoloMonti42) [reply sys/build.sh]: we don't, the script terminates at the beginning if the cert is already present
```
security find-certificate -Z -p -c "org.radare.radare2" /Library/Keychains/System.keychain > /dev/null 2>&1
 EXIT_CODE=$?
 if [ $EXIT_CODE -eq 0 ]; then
 	# Certificate has already been generated and installed
 	exit 0
 fi
```

**Comment by radare** [comment]
good!

**Pull request merged**